### PR TITLE
Feat/#66 styling

### PR DESCRIPTION
**Pull request by ciaranschutte**

### CODE REVIEW

File: website/src/components/TreeView/DictionaryTree.js [hunk 1]
```diff
@@ -209,7 +209,7 @@ const FileNode = ({ fileDef, onExpandStateChange }) => {
         className={css`
           &::before,
           &::after {
-            left: -4px !important;
+            _left: -4px !important;
```
Comment (hlminh2000) [inline website/src/components/TreeView/DictionaryTree.js]: what's `_left`?

Comment (ciaranschutte) [reply website/src/components/TreeView/DictionaryTree.js]: meant to remove this, thanks @hlminh2000

Comment (ciaranschutte) [reply website/src/components/TreeView/DictionaryTree.js]: sometimes use the `_` to make it invalid instead of deleting/comment :)

Comment (hlminh2000) [reply website/src/components/TreeView/DictionaryTree.js]: lol I was almost excited to learn some new css :P

Comment (ciaranschutte) [reply website/src/components/TreeView/DictionaryTree.js]: fixed @hlminh2000

File: website/src/components/TreeView/DictionaryTree.js [hunk 2]
```diff
@@ -94,11 +94,11 @@ const FileDisplayBox = ({ fileName, required = false, fields, onExpandStateChang
...
   const collapseAllMessenger = useExpandStateMessenger();
   React.useEffect(() => {
-    const onDisatch = ({ expanded }) => {
+    const onDispatch = ({ expanded }) => {
       setExpanded(expanded);
     };
-    collapseAllMessenger.subscribe(onDisatch);
-    return () => collapseAllMessenger.unsubscribe(onDisatch);
+    collapseAllMessenger.subscribe(onDispatch);
+    return () => collapseAllMessenger.unsubscribe(onDispatch);
```
Comment (hlminh2000) [inline website/src/components/TreeView/DictionaryTree.js]: thank you for cleaning up my mess!!

**Pull request merged**